packages/flutter_tools/test/integration.shard/flutter_plugins_dependencies_test.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+import 'package:file/file.dart';
+
+import '../src/common.dart';
+import 'test_utils.dart';
+
+void main() {
+  late Directory tempDir;
+
+  setUp(() async {
+    tempDir =
+        createResolvedTempDirectorySync('flutter_plugins_dependencies_test.');
+  });
+
+  tearDown(() async {
+    tryToDelete(tempDir);
+  });
+
+  test(
+      '.flutter-plugins-dependencies correctly denotes project dev dependencies on iOS, Android',
+      () async {
+    // Create Flutter project.
+    final String flutterBin =
+        fileSystem.path.join(getFlutterRoot(), 'bin', 'flutter');
+
+    await processManager.run(<String>[
+      flutterBin,
+      'create',
+      tempDir.path,
+      '--project-name=testapp',
+    ], workingDirectory: tempDir.path);
+
+    final File pubspecFile = tempDir.childFile('pubspec.yaml');
+    expect(pubspecFile.existsSync(), true);
```

Comment:
nit: Try using the `exists` matcher instead (there should be other examples of this in the flutter_tools/test directory, look for `package:file_testing` I believe?).

Reply:
I couldn't find such a matcher. There are convenience methods I've seen but they just call this or similar methods.

Reply:
<https://pub.dev/documentation/file_testing/latest/file_testing/exists-constant.html>

No worries, this isn't critical. Thanks!